src/styles/index.scss

```diff
@@ -1,3 +1,9 @@
-body {
+@import "./utils/variables";
+@import "card.scss";
+@import "stars.scss";
+
+* {
```

Comment:
only `box-sizing: border-box;` rule is allowed to use for `*` selector
move `font-family: Roboto, Arial, Helvetica, sans-serif;` to `html` selector
move `margin: 0` to `body`, `h2`, `p` etc